tasks/sep/007-fpac-upgrade/README.md

```diff
@@ -0,0 +1,114 @@
+  - After: `0x0000000000000000000000001eb2ffc903729a0f03966b917003800b145f56e2`
+  - Meaning: The `DisputeGameFactory` owner has been updated to the new Sepolia safe (`0x1Eb2fFc903729a0F03966B917003800b145F56E2`).
+- Key `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  - Before: `0x0000000000000000000000001b3cfc1f692076538c9b53750ebc412042d48605`
+  - After: `0x000000000000000000000000c658fddbfbe85b59c9fdabb9e78598967e33cb96`
+  - Meaning: The implementation of the `DisputeGameFactory` proxy has been upgraded to `0xc658fddbfbe85b59c9fdabb9e78598967e33cb96`
+- Key `0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e`
+  - Before: `0x00000000000000000000000054a16603469dc1f4bf7197908067d858d04864e5`
+  - After: `0x000000000000000000000000dae97900d4b184c5d2012dcdb658c008966466dd`
+  - Meaning: The implementation of the `PermissionedDisputeGame` has been upgraded to `0xdaE97900D4B184c5D2012dcdB658c008966466DD`
+- Key `0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b`
+  - Before: `0x000000000000000000000000a2f42e0a3571e14a831bc5ea58cdf8b98d9b11f2`
+  - After: `0x000000000000000000000000c7f2cf4845c6db0e1a1e91ed41bcd0fcc1b0e141`
+  - Meaning: The implementation of the `FaultDisputeGame` has been upgraded to `0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141`
+
+**[`0x16Fc5058F25648194471939df75CF27A2fdC48BC`][portal-prox-etherscan] (The `OptimismPortal` proxy contract)**
+
+The transaction should only result in one changed storage slot in the `OptimismPortal` proxy contract:
+
+- Key `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  - Before: `0x000000000000000000000000215b3a2d1766616d1c4701f3d117348711baae93`
+  - After: `0x00000000000000000000000062749b56f045d30e1971ffa9880d496c701eb386`
+  - Meaning: EIP-1967 Implementation slot, updated to the new `OptimismPortal2` implementation address.
+
+**[`0xf64bc17485f0b4ea5f06a96514182fc4cb561977`][mock-council-safe-etherscan] (The mock Council Gnosis Safe)**
+
+- Key `0x0000000000000000000000000000000000000000000000000000000000000005`
+  - Before: `0x0000000000000000000000000000000000000000000000000000000000000001`
+  - After: `0x0000000000000000000000000000000000000000000000000000000000000002`
+  - Meaning: The nonce of the safe has been bumped by `1`.
+
+**[`0x1eb2ffc903729a0f03966b917003800b145f56e2`][upgrade-safe-etherscan] (The upgrade Gnosis safe)**
+
+- Key `0x0000000000000000000000000000000000000000000000000000000000000005`
+  - Before: `0x0000000000000000000000000000000000000000000000000000000000000001`
+  - After: `0x0000000000000000000000000000000000000000000000000000000000000002`
+  - Meaning: The nonce of the safe has been bumped by `1`.
+- Key `0x8e101d76327e9b6065ed1302ba375078344b4376846d9948cae0d1aa2d8c630e`
+  - Before: `0x0000000000000000000000000000000000000000000000000000000000000000`
+  - After: `0x0000000000000000000000000000000000000000000000000000000000000001`
+  - Meaning: ????
```

Comment:
Not sure what this is; Probably a mapping key.

Reply:
the only mappings in the safe are `signedMessages` and `approvedHashes`, with only the latter in use. I'm guessing this is an update to the latter based on https://github.com/safe-global/safe-smart-account/blob/main/contracts/Safe.sol#L355

Reply:
[Here](https://github.com/ethereum-optimism/superchain-ops/blob/cl/fpac-2/tasks/eth/004-add-superchainConfig/Validation.md#L105-L120) is an example of how we have handled identifying mapping keys in the past using `cast index`.